src/Orleans/Streams/QueueAdapters/IQueueAdapterReceiver.cs

```diff
@@ -32,8 +32,6 @@ namespace Orleans.Streams
     /// </summary>
     public interface IQueueAdapterReceiver
     {
-        QueueId Id { get; }
```

Comment:
I can maybe understand why QueueId is not a core functionality, but please do explain. It is a **queue** adapter receiver, a receiver for a **QUEUE**, so it is not immediately obvious why some way to identify a queue, like QueueId, is not a core functionality.